markup.go

```diff
@@ -47,36 +47,32 @@ type Event struct {
 
 // MarkupOrComponentOrHTML represents one of:
 //
-//  Markup
+//  MarkupList
```

Comment:
Couldn't this be `Applyer` instead?

Reply:
See https://github.com/gopherjs/vecty/pull/134#discussion_r135370011